src/main/kotlin/com/github/platan/tests_execution_chart/reporters/html/HtmlTableFormatter.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.platan.tests_execution_chart.reporters.html
+
+import com.github.platan.tests_execution_chart.TestExecutionScheduleReport
+
+class HtmlTableFormatter {
+
+    fun format(report: TestExecutionScheduleReport): String {
+        val tbody = StringBuilder()
+        report.results.sortedByDescending { it.durationMs }.map {
+            tbody.append(
+                """<tr><td>${it.testName}</td>
```

Comment:
HTML special characters in testName/className values should be escaped. Currently test name `def "test 1 <tr>"() {` results with broken table:
<img width="1392" alt="Zrzut ekranu 2022-10-24 o 19 00 31" src="https://user-images.githubusercontent.com/1526000/197583611-a4bbec70-2e40-4899-b86a-4ff167f8b985.png">

Reply:
I've added escaping for html and test for checking </> characters.

Reply:
Thanks for fixing and adding test for this case.